src/expressions/if-expr.md

```diff
@@ -126,30 +104,43 @@ if let E::X(n) | E::Y(n) = v {
 ```
 
 The expression cannot be a [lazy boolean operator expression][_LazyBooleanOperatorExpression_].
-Use of a lazy boolean operator is ambiguous with a planned feature change of the language (the implementation of if-let chains - see [eRFC 2947][_eRFCIfLetChain_]).
```

Comment:
Could this also retain the old note about how to deal with use of lazy boolean expressions when combined with `if let`?  Maybe something like:

> The scrutinee expression cannot be a [lazy boolean operator expression] due to ambiguity and precedence with [chains of expressions].
> If a lazy boolean operator expression is needed, then parentheses can be used.  For example:
>
> ```rust
> if let PAT = ( EXPR && EXPR ) { .. }
> if let PAT = ( EXPR || EXPR ) { .. }
> ```

Reply:
Done without invalid snippets